### ISSUE TITLE
create another view vw_eml_associatedParty

### ISSUE THREAD

**Issue by mobb** [lter/LTER-core-metabase#24]
see the original MCR metabase view.

**Comment by mobb** [comment]
see commit https://github.com/lter/LTER-core-metabase/commit/2bff7e019f710023fdd58b65ee0f8b9a6534952c

@atn38 - you should be able to run this script in the sql pane for the database. Just like the other scripts, this one is technically a 'template', you have to edit the name of the database owner in. with just the the sample datasets, there should be 3 rows in the view.

**Comment by atn38** [comment]
Thanks Margaret!

Looking quickly, is it correct that this would exclude creators, contact,
publisher, and metadataProvider?

The middle two is in boilerplate, but I don't think the last one is.

**Comment by mobb** [comment]
chat moved to slack.  summary: metabase (lter_metabase schema) can hold content for any of the responsibleParty elements. 
this view excludes people or orgs with roles that have explicit dataset-level EML elements: ie, creator, publisher, metadataProvider, contact
however, there is no controlled vocabulary for roles, its by convention only.

question about boilerplate pertains to how R-code gets content. and yes, the workflow is anticipated to rely on some boilerplate (xml).

**Comment by atn38** [comment]
I'm ok with putting those three roles in boilerplate, yesterday only wanted to clarify your intent.

Other issues I noticed: 

1. vw_associatedparty contains onlineurl (webpage) column, which vw_creator doesn't have.
Makes more sense for creators (likely PIs and profs) to have a page than others (likely students).
now that we are on this topic, is webpage url info something we'd like to maintain? Online links change and break a lot. 

2. vice versa: vw_creator has ORCID ids while vw_associatedparty doesn't.
I'd argue for vw_associatedparty to also have orcid as a column. I can see some personnel not having ORCIDs (students, field technicians), but it's good to have the option to include ORCIDs in EML for those that do.

**Comment by mobb** [comment]
so sorry - that is an artifact of my merging 3 databases! 
completely agree - those fields make sense. 

what I did, see commit https://github.com/lter/LTER-core-metabase/commit/d9aa6c1b9a517a0f8f06c8b03f94041b50e1d14c
1. select matches statement from vw_eml_creator (order as well as fields)
2. also note, that I took this opportunity to rename the files with a prefixed "0" so scripts will continue to sort in order if we go over 10. (and moved script 05* to the correct dir!)

@atn38 , when you run this script, you will probably have to drop the existing view before you create the new one. to do that, just uncomment the statement 
"-- DROP VIEW mb2eml_r.vw_eml_associatedparty; 
In my instance, postgres would not recreate the view with columns rearranged.
